hack/validate/deprecate-crioctl

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+# Check that no new tests are being added to integration-cli
```

Comment:
This comment needs to be updated as well.

Reply:
fixed